bears/python/VultureBear.py

```diff
@@ -32,14 +63,10 @@ def run(self):
 
         See <https://bitbucket.org/jendrikseipp/vulture> for more information.
         """
-        stdout_output, _ = run_shell_command(
-            (self.EXECUTABLE,) +
-            tuple(filename for filename in self.file_dict.keys()),
-            cwd=self.get_config_dir())
-
-        for match in re.finditer(self.OUTPUT_REGEX, stdout_output):
-            groups = match.groupdict()
+        filenames = list(self.file_dict.keys())
+        unused = retrieve_unused(filenames)
+        for unused_entity in unused:
```

Comment:
TODO: Since unused imports are detected with 100% accuracy, implement a patch yielding result which automatically removes the statement.